File: RepoTransBench/target_projects/JavaScript/Python/foradian_fluxday/tests/original/test_sample.py
import pytest
from src.sample import add, max

class TestAdd:
    def test_adds_two_positive_numbers(self):
        assert add(2, 4) == 6

    def test_adds_a_negative_and_positive_number(self):
        assert add(-5, 3) == -2

    def test_adds_two_zeros(self):
        assert add(0, 0) == 0

class TestMax:
    def test_maximum_in_positive_array(self):
        assert max([1, 3, 2]) == 3

    def test_maximum_in_negative_array(self):
        assert max([-1, -3, -2]) == -1

    def test_returns_none_for_empty_array(self):
        assert max([]) is None

    def test_returns_none_for_undefined(self):
        assert max(None) is None

    def test_returns_value_for_single_element_array(self):
        assert max([42]) == 42

File: RepoTransBench/target_projects/C/Rust/libav_c99-to-c89/tests/test_sample.rs
use sample::{add, max, is_even};

#[test]
fn test_add() {
    assert_eq!(add(2, 3), 5);
    assert_eq!(add(-1, 1), 0);
    assert_eq!(add(0, 0), 0);
    assert_eq!(add(-5, -6), -11);
}

#[test]
fn test_max() {
    assert_eq!(max(2, 3), 3);
    assert_eq!(max(5, 1), 5);
    assert_eq!(max(-1, -6), -1);
    assert_eq!(max(7, 7), 7);
}

#[test]
fn test_is_even() {
    assert_eq!(is_even(2), true);
    assert_eq!(is_even(3), false);
    assert_eq!(is_even(0), true);
    assert_eq!(is_even(-4), true);
    assert_eq!(is_even(-5), false);
}